code/components/citizen-devtools/src/ResourceTimeWarnings.cpp

```diff
@@ -67,7 +69,67 @@ constexpr auto tuple_slice(Cont&& t)
 	return detail::slice_impl<I1>(std::forward<Cont>(t),
 	std::make_index_sequence<I2 - I1>{});
 }
+
+static std::chrono::microseconds lastHitch;
+
+#ifdef GTA_FIVE

```

Comment:
This does not affect RDR3 as it does off-tread windowing, and NY may use a different loop on render anyway.